multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -64,51 +66,14 @@ fn verify_bill_payment() {
         "99998043204000000000",
         "472000000000000000000000000",
     );
-    let consuming_config = NodeStartupConfigBuilder::standard()
-        .blockchain_service_url(blockchain_server.service_url())
-        .chain("dev")
-        .consuming_wallet_info(ConsumingWalletInfo::PrivateKey(contract_owner_secret_key))
-        .earning_wallet_info(EarningWalletInfo::Address(format!(
-            "{}",
-            contract_owner_wallet.clone()
-        )))
-        .build();
+    let (consuming_config, _) = build_config(&blockchain_server, &seed, deriv_path);
 
-    let (serving_node_1_wallet, serving_node_1_secret) =
-        make_node_wallet(&seed, derivation_path(0, 1).as_str());
-    let serving_node_1_config = NodeStartupConfigBuilder::standard()
-        .blockchain_service_url(blockchain_server.service_url())
-        .chain("dev")
-        .consuming_wallet_info(ConsumingWalletInfo::PrivateKey(serving_node_1_secret))
-        .earning_wallet_info(EarningWalletInfo::Address(format!(
-            "{}",
-            serving_node_1_wallet.clone()
-        )))
-        .build();
-
-    let (serving_node_2_wallet, serving_node_2_secret) =
-        make_node_wallet(&seed, "m/44'/60'/0'/0/2");
-    let serving_node_2_config = NodeStartupConfigBuilder::standard()
-        .blockchain_service_url(blockchain_server.service_url())
-        .chain("dev")
-        .consuming_wallet_info(ConsumingWalletInfo::PrivateKey(serving_node_2_secret))
-        .earning_wallet_info(EarningWalletInfo::Address(format!(
-            "{}",
-            serving_node_2_wallet.clone()
-        )))
-        .build();
-
-    let (serving_node_3_wallet, serving_node_3_secret) =
-        make_node_wallet(&seed, "m/44'/60'/0'/0/3");
-    let serving_node_3_config = NodeStartupConfigBuilder::standard()
-        .blockchain_service_url(blockchain_server.service_url())
-        .chain("dev")
-        .consuming_wallet_info(ConsumingWalletInfo::PrivateKey(serving_node_3_secret))
-        .earning_wallet_info(EarningWalletInfo::Address(format!(
-            "{}",
-            serving_node_3_wallet.clone()
-        )))
-        .build();
+    let (serving_node_1_config, serving_node_1_wallet) =
+        build_config(&blockchain_server, &seed, derivation_path(0, 1));
+    let (serving_node_2_config, serving_node_2_wallet) =
+        build_config(&blockchain_server, &seed, derivation_path(0, 2));
+    let (serving_node_3_config, serving_node_3_wallet) =
+        build_config(&blockchain_server, &seed, derivation_path(0, 3));
 
     let amount = 10u64
         * u64::try_from(node_lib::accountant::PAYMENT_CURVES.permanent_debt_allowed_gwub).unwrap();
```

Comment:
`gwub` is actually short for `$SUB in Gwei` so if you felt so inclined to change it to `gwei` or `qwaq` or something else that would be fine. Or you could leave it too... that's fine. It's one of those 'you had to be there to know' kind of things.

Reply:
I think we will likely change the naming of this sometime soon - we are looking to change the naming of the payment curve values anyways, since they are very confusing!

Reply:
`gwasq` ?

Reply:
I'm going to take care of the nomenclature in a different card because I've started that in there and I want to do it just at one place.